powershell/public/cisa/exchange/Test-MtCisaDmarcRecordExist.ps1

```diff
@@ -35,7 +35,7 @@ function Test-MtCisaDmarcRecordExist {
 
     $seen = @{}
     $dmarcRecords = @()
-    foreach($domain in $acceptedDomains){
+    foreach($domain in ($acceptedDomains | Get-Unique -AsString)){
```

Comment:
The `Get-Unique` cmdlet "requires" items in the list to be sorted for it to work as expected, and is also case-sensitive. Using `Sort-Object -Unique` may provide the desired outcome, as it sorts and is case-insensitive by default.